src/edu/csus/ecs/pc2/core/StringUtilities.java

```diff
@@ -276,4 +276,25 @@ public static boolean getBooleanValue(String string, boolean defaultBoolean) {
         return value;
     }
     
+    /**
+     * get team number from login string.
+     * 
+     * @param user,
+     *            team login in form team#, team102
+     * @return null if no number found or number
```

Comment:
The above comment doesn't make sense; please explain what "if no number found or number" is supposed to mean.

Reply:
Changed to: @return null if no team number found or string after team is not a number.

Reply:
pushed